stewards.json

```diff
@@ -126,5 +126,9 @@
     {
         "ca": "Golden_Ratio_Staking/server_ca.crt",
         "endpoint": "https://sommelier.goldenratiostaking.net:5734"
-    }
+    },
+    {
+        "ca": "frenchchocolatine/server_ca.crt",
+        "endpoint": "https://sommelier.frenchchocolatine.com:5734"
```

Comment:
Your domain here is "sommelier.frenchchocolatine.com" but the CN in your certificate is simply "frenchchocolatine", I believe we need these to match. If you don't mind, please regenerate your certificate with the full domain name in the Common Name field.